src/jobs/test-nut.yml

```diff
@@ -67,15 +67,17 @@ steps:
       steps:
         - restore_cache:
             keys:
+              - v3-npm-{{ CIRCLE_BRANCH }}-{{checksum "yarn.lock"}}
               - v3-npm-{{checksum "yarn.lock"}}
```

Comment:
I still think this line wants to be `- v3-npm-{{ CIRCLE_BRANCH }}`

Reply:
@amphro 
This was the keys you called out in slack
` We should change it to branch-yarnlock, then yarnlock, then just npm`